charts/customer-center/README.md

```diff
@@ -0,0 +1,158 @@
+| frontend.livenessProbe.enabled | bool | `true` | Enable liveness probe on frontend |
+| frontend.livenessProbe.failureThreshold | int | `6` | Number of tries to perform the probe |
+| frontend.livenessProbe.initialDelaySeconds | int | `60` | Number of seconds after the container has started before liveness probe is initiated |
+| frontend.livenessProbe.periodSeconds | int | `10` | How often (in seconds) to perform the probe |
+| frontend.livenessProbe.successThreshold | int | `1` | Minimum consecutive successes for the probe to be considered successful after having failed |
+| frontend.livenessProbe.timeoutSeconds | int | `5` | Number of seconds after which the probe times out |
+| frontend.readinessProbe.enabled | bool | `true` | Enable readiness probe on frontend |
+| frontend.readinessProbe.failureThreshold | int | `6` | Number of tries to perform the probe |
+| frontend.readinessProbe.initialDelaySeconds | int | `30` | Number of seconds after the container has started before readiness probe is initiated |
+| frontend.readinessProbe.periodSeconds | int | `10` | How often (in seconds) to perform the probe |
+| frontend.readinessProbe.successThreshold | int | `1` | Minimum consecutive successes for the probe to be considered successful after having failed |
+| frontend.readinessProbe.timeoutSeconds | int | `5` | Number of seconds after which the probe times out |
+| frontend.replicaCount | int | `1` | (int) Number of Backend replicas |
+| frontend.resources | object | `{}` | Resource limits for frontend |
+| frontend.service.externalPort | int | `80` | External Port of frontend service |
+| frontend.service.internalPort | int | `80` | Internal Port of frontend service |
+| frontend.service.name | string | `"customer-center-frontend"` | Frontend service name |
+| frontend.service.type | string | `"ClusterIP"` | Frontend service type |
+| ingress.annotations | object | `{}` | Ingress annotations |
+| ingress.certificate.certFilename | string | `"tls.crt"` | Cert file name |
+| ingress.certificate.certKeyFilename | string | `"tls.key"` | Cert key filename |
+| ingress.certificate.dnsNames[0] | string | `"my.example.com"` |  |
+| ingress.certificate.issuerRef.group | string | `"cert-manager.io"` |  |
+| ingress.certificate.issuerRef.kind | string | `"ClusterIssuer"` |  |
+| ingress.certificate.issuerRef.name | string | `"letsencrypt-prod"` |  |
+| ingress.certificate.name | string | `"customer-center"` | Name of certificate |
+| ingress.certificate.secret | string | `"customer-center-tls"` | Name of TLS secret |
+| ingress.enabled | bool | `true` | Enable ingress for customer-center |
+| ingress.hosts | list | `["my.example.com"]` | Ingress hostnames |
+| ingress.tls | list | `[{"hosts":["my.example.com"],"secretName":"customer-center-tls"}]` | Ingress TLS options |
+| postgresql.enabled | bool | `true` | Enable PostgreSQL for persistence |
```

Comment:
Are these not just shadowing the values from the upstream postgresql chart?

Reply:
No not even, and I don't know why the ingress part has been added. I will remove it and just keep the switch to enable postgresql.